token-lending/program/src/processor.rs

```diff
@@ -1577,6 +1585,42 @@ fn process_liquidate_obligation(
     let clock = &Clock::from_account_info(next_account_info(account_info_iter)?)?;
     let token_program_id = next_account_info(account_info_iter)?;
 
+    let _withdraw_collateral_amount = _liquidate_obligation(
```

Comment:
I don't think you need to have the `let_withdraw_collateral_amount`

Reply:
ah yeh forgot to remove this will do that